config/settings/development.yml

```diff
@@ -0,0 +1,3 @@
+valid_purl_url: <%= /http?:\/\/localhost:3000\/*/ %>
```

Comment:
Why is this in erb?

Reply:
Based on https://github.com/sul-dlss/sul-embed/blob/master/config/settings.yml#L6

Reply:
Let's not perpetuate bad choices. Here's a fix https://github.com/sul-dlss/sul-embed/pull/1423

Reply:
This value is no longer used in the code. Please remove it.